packages/nimble-components/src/drawer/tests/drawer.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('Drawer', () => {
     async function completeAnimationAsync(
         nimbleDrawerElement: Drawer | Drawer<string>
     ): Promise<void> {
-        return new Promise(resolve => {
+        await new Promise<void>(resolve => {
```

Comment:
There were a couple spots that failed type checking after running the formatter like here. Because the return of this function is `Promise<void>` that means no return value value is needed anymore. The fixer changed this to just `await new Promise()` (which I think is a better representation of this function to not return anything since it's `Promise<void>`) 

But now that it is not part of the return type, the Promise return value cannot be inferred and had to be made explicit.